dist/tools/openocd/openocd.sh

```diff
@@ -191,8 +191,16 @@ do_debug() {
     test_elffile
     test_ports
     test_tui
+    # setsid is needed so that Ctrl+C in GDB doesn't kill OpenOCD
+    [ -n "$(which setsid)" ] && SETSID=setsid
```

Comment:
Actually, it makes more sense to check whether the SETSID variable is null. (sorry for commenting in the commit before)
```sh
[ -n "${SETSID}" ] && SETSID=$(which setsid)
```